File: CPP/Program/Product.cpp
// +==========================================================+
// +                                                          +
// + Saya Raden Rahman Ismail (2200311)                       +
// + mengerjakan soal Latihan dalam mata kuliah DPBO          +
// + untuk keberkahanNya maka saya tidak melakukan kecurangan +
// + seperti yang telah dispesikasikan.Aamiin                 +
// +                                                          +
// +==========================================================+

// memanggil libary yang digunakan
#include <iostream>
#include <string>

// using standard namespace
using namespace std;

// class Product
class Product
{
private:
    string idProduct;
    string name;
    string brand;
    string price;

public:
    Product()
    {
        this->idProduct = "";
        this->name = "";
        this->brand = "";
        this->price = "";
    };

    Product(string idProduct, string name, string brand, string price)
    {
        this->idProduct = idProduct;
        this->name = name;
        this->brand = brand;
        this->price = price;
    }

    // Fungsi get
    string getidProduct()
    {
        return this->idProduct;
    }
    string getName()
    {
        return this->name;
    }
    string getBrand()
    {
        return this->brand;
    }
    string getPrice()
    {
        return this->price;
    }

    // fungsi set
    void setidProduct(string idProduct)
    {
        this->idProduct = idProduct;
    }
    void setName(string name)
    {
        this->name = name;
    }
    void setBrand(string brand)
    {
        this->brand = brand;
    }
    void setPrice(string price)
    {
        this->price = price;
    }

    ~Product()
    {
    }
};

File: PYTHON/Program/Product.py
# membuat class product
class Product:
    __idProduct = ""
    __name = ""
    __brand = ""
    __price = ""

    def __init__(self, idProduct="", name="", brand="", price=""):
        self.__idProduct = idProduct
        self.__name = name
        self.__brand = brand
        self.__price = price

    # Fungsi get
    def get_idProduct(self):
        return self.__idProduct

    def get_name(self):
        return self.__name

    def get_brand(self):
        return self.__brand

    def get_price(self):
        return self.__price

    # Fungsi set
    def set_idProduct(self, idProduct):
        self.__idProduct = idProduct

    def set_name(self, name):
        self.__name = name

    def set_brand(self, brand):
        self.__brand = brand

    def set_price(self, price):
        self.__price = price

